activities/index.md

```diff
@@ -6,10 +6,6 @@ type: Index
 
 # Activities
 
-Higher level activities of the Foundation for Public Code that directly serve to further the [mission](../organization/mission.md):
```

Comment:
This explainer is still true, right? And it feels helpful as it explains why we haven't an activity for every little thing we do.

Reply:
Yes, restored to original. I'm wondering how process code will fit in here, and whether standard maintenance is not also a higher level activity.

My instinct was to flatten it for staff - as each staff member has their own priority, but fine to leave as is.